src/ControlTasks/read_full_camera.py

```diff
@@ -0,0 +1,55 @@
+from .control_task_base import ControlTaskBase
+
+import cv2
+import depthai as dai
+
+
+class ReadFullCamera(ControlTaskBase):
```

Comment:
What is the difference between this and read_camera.py?

Reply:
They read different types of cameras. `read_full_camera` is specifically for the oak-d camera we ordered which requires different processing. 

But you bring up a good point that the names could be improved